content/en/docs/collector/deployment/gateway/index.md

```diff
@@ -198,6 +199,30 @@ The load-balancing exporter emits metrics including
 that you can use for health and performance monitoring of the OTLP endpoint
 collector.
 
+## Typical Gateway deployment architecture
+
+The following diagram shows a typical deployment architecture for a gateway
+![gateway](otel-gateway-arch.svg)
+
+There are a few limitations in running the OTel collector in gateway mode.
+
+- Few receivers need to be unique per host instance. Running multiple instances
+  of these receivers will result in Duplicate data. It is recommended not to use
+  them in the Gateway collector but only for the collector running in Daemonset
```

Comment:
```suggestion
  them in the Gateway Collector but only for the Collector running in Daemonset
```

Have we introduced "Daeminset mode" somewhere? This also sounds k8s specific, so we need to call out that this is a k8s specific example somewhere at the top